docs/source/release/v3.10.0/muon.rst

```diff
@@ -17,6 +17,10 @@ Algorithms
 Fit Functions
 -------------
 
+Bugfixes
+--------
+- Fixed a bug that menat transverse field asymmetry data was normalized to bin width. 
```

Comment:
I placed it here because the bug fix was specific to the muon interface. I can move it if you think that is better.